general/apero_reduction_interface/simple_ari.py

```diff
@@ -2865,7 +2866,7 @@ def add_obj_pages(gsettings: dict, settings: dict, profile: dict,
             # run the pool
             results = add_obj_page(*itargs)
             # push result to result storage
-            results_dict[key] = results
+            results_dict[key] = results[it]
```

Comment:
This fixes a bug with the serial (`n_cores == 1`) case. Otherwise the value in `results_dict` will have an extra layer, e.g. `{"OBJNAME": {it: {"OBJURL": "something"}}}` instead of `{"OBJNAME": {"OBJURL": "something"}}`.